script/playbackstrategy/msestrategy.js

```diff
@@ -373,6 +373,7 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
         }
 
         if (windowType === WindowTypes.SLIDING) {
+          DebugTool.keyValue({key: 'initial-playback-time', value: startTime});
```

Comment:
Again, probably best to keep the log the same.

Reply:
it is.

Reply:
The time passed into dashjs is parsed, the log isn't. i know it will be the same value just with decimal places. As there is a reason we parse the value.

Reply:
I get you now!